clang/include/clang/AST/RecursiveASTVisitor.h

```diff
@@ -2154,6 +2144,15 @@ bool RecursiveASTVisitor<Derived>::TraverseFunctionHelper(FunctionDecl *D) {
                                                   TALI->NumTemplateArgs));
       }
     }
+    // FIXME: Do we want to traverse the explicit template arguments for
```

Comment:
I originally added this comment because it changes behavior (i.e. explicit template arguments for dependent friends were not previously traversed, but are traversed with this change). Upon a second examination, I don't think the comment is necessary, and the correct behavior is to traverse the explicit arguments (as we would traverse them for a non-dependent specialization).